train_gridworld.py

```diff
@@ -0,0 +1,62 @@
+import random
+import numpy as np
+import matplotlib.pyplot as plt
+from discrete_pendulum import Pendulum
+# plt.style.use('seaborn')
+import gridworld
+import model_based
+import model_free
+
```

Comment:
All the plots and models seem to be working well. And they're easy to understand/don't seem slow.